tests/unit/infrastructure/s3/s3Service.test.ts

```diff
@@ -0,0 +1,69 @@
+import { S3 } from 'aws-sdk';
+import getFromS3 from '../../../../src/infrastructure/s3/s3Service';
+
+describe('S3 Service', () => {
+  beforeAll(async (done) => {
```

Comment:
I'm not clear on what this callback is doing 🤔  Would `jest.resetModules()` or [this SO question](https://stackoverflow.com/questions/48033841/test-process-env-with-jest) help?

Reply:
Oops, hangover from my trying a different mocking idea.